src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/integration/IntegrationTestBase.kt

```diff
@@ -33,4 +33,4 @@ abstract class IntegrationTestBase {
 }
 
 fun Offender.latestBooking(): OffenderBooking =
-  this.bookings.firstOrNull { it.active } ?: throw IllegalStateException("Offender has no active bookings")
+  this.getAllBookings()?.firstOrNull { it.active } ?: throw IllegalStateException("Offender has no active bookings")
```

Comment:
I wonder if we should move `Offender.latestBooking()` to the `Offender` class instead of as an extension function? I suspect it's by far the most used.

Reply:
Good idea